wallet-confs/stakecubecoin--v3.0.1.conf

```diff
@@ -3,8 +3,8 @@ listen=1
 rpcuser=
 rpcpassword=
 rpcallowip=0.0.0.0/0
-port=26969
-rpcport=26970
+port=40000
+rpcport=39999
 txindex=1
```

Comment:
I don't see a problem here, but I'm just curious about workflow. It says `wallet-confs/stakecubecoin--v3.0.1.conf` was created by renaming `wallet-confs/lux--v5.3.3.conf` to `wallet-confs/stakecubecoin--v3.0.1.conf` and then modifying it. Shouldn't `wallet-confs/stakecubecoin--v3.0.1.conf` have been created by running `app.py --coins STAKECUBECOIN` ? Or maybe that's what you did and Git just got confused because you had also recently deleted `wallet-confs/lux--v5.3.3.conf`?

Reply:
Yes, Git got confused by all the stuff going on. It had pretty low confidence on some of the descriptions it applied to the changes (eg: 59%, 67%)